src/chia_plot.cpp

```diff
@@ -344,6 +346,10 @@ int main(int argc, char** argv)
 			break;
 		}
 		std::cout << "Crafting plot " << i+1 << " out of " << num_plots << std::endl;
+		if (tmptoggle && (i % 2 == 1)) {
+			tmp_dir.swap(tmp_dir2);
```

Comment:
this should be at the end of the loop, without any `if()`, if I understand your goal correctly

Reply:
I think he is using tmptoggle as a variable for the users to set the toggle on and off. the "if" should be there in this case. I'd not activate it, and other people that use RAMDISK would also not turn this on.

Reply:
ah yeah, I mean without the `&& (i % 2 == 1)`